IFC4x3/Properties/s/SignalPostTelephoneID_0NCWcMrUb2H8fwqgBRgcB0/Documentation.md

```diff
@@ -0,0 +1 @@
+Reference to the signal post telephone.
```

Comment:
In my opinion, references should be in the model.

Reply:
@czapplitec and @fhulin can this be a reference in the model?

Reply:
This is just information needed. The post telephone is not modelled as an element. In reality, it is a device attached to signal post. At far as I know, it is quite old technology in signalling that is not used anymore in newly constructed railways.